### DISCUSSION TITLE
require some lua file from a luaj read lua file

### DISCUSSION THREAD

**Discussion by Tofaa2** [gudzpoz/luajava#190] [Q&A]
I have a luaj project, and a file that's being read by Lua#load(), however that lua file has a require on the top that requires another lua file in the same project directory, I cannot seem to access it from my luaj loaded lua file. Is there a way around this, im not sure how to approach it

**Comment by gudzpoz** [comment]
[Lua::setExternalLoader](https://gudzpoz.github.io/luajava/javadoc/party/iroiro/luajava/Lua.html#setExternalLoader(party.iroiro.luajava.ExternalLoader)) is intended for this and should be able to address your needs. There is also a built-in ExternalLoader implementation, [ClassPathLoader](https://gudzpoz.github.io/luajava/javadoc/party/iroiro/luajava/ClassPathLoader.html), that loads Lua modules from Java classpath.
